packages/cli/src/commands/replay/replay.command.ts

```diff
@@ -184,6 +192,7 @@ export const replayCommandHandler: (
 
   await eventsFinishedPromise;
   await writesFinishedPromise;
+  server?.closeServer?.();
```

Comment:
```suggestion
  server?.closeServer();
```

I think, since the method exists when the server does.